lib/fluent/plugin/in_tail.rb

```diff
@@ -494,9 +500,9 @@ def on_notify
             begin
               while true
                 if @buffer.empty?
-                  @io.read_nonblock(2048, @buffer)
+                  @io.readpartial(2048, @buffer)
```

Comment:
> (Won't happen such case in current use case?)

In in_tail case, `read_nonblock` always raises EOFError. Is this unexpected with `read_nonblock` or ok with local file?